### ISSUE TITLE
Rendering of rack reservations is poor

### ISSUE THREAD

**Issue by candlerb** [netbox-community/netbox#11086]
### NetBox version

v3.3.9

### Python version

3.8

### Steps to Reproduce

1. Create a rack.  (Any rack; doesn't matter if it has devices in it or not)
2. Create a 1U reservation
3. Render the rack elevation

![image](https://user-images.githubusercontent.com/44789/205256804-51287958-6036-4e1f-b33e-b8d6c01a0f84.png)

### Expected Behavior

In older versions of Netbox, it used to be the case that the reservation area was hatched out.

### Observed Behavior

It's very weird: a small piece of hatching sits *to the side* of the rack.

![image](https://user-images.githubusercontent.com/44789/205256670-d561f339-2d89-4cc8-a471-1e2b2d45a7d3.png)

This is viewed using Chrome 107 under macOS 12.6.1.

(This has actually been broken for a while and I don't remember when it first started happening, but I've only just got around to reporting it)

**Comment by candlerb** [comment]
Another example: new 42U rack, no devices, reservation at position 3.

![image](https://user-images.githubusercontent.com/44789/205258081-3866e9fc-639d-41e3-8b27-c27c6587fa60.png)

**Comment by candlerb** [comment]
Hmm, so checking in the source code, this is intentional:

```
    def draw_margin(self):
        """
        Draw any rack reservations in the right-hand margin alongside the rack elevation.
        """
```

In that case, I think the chosen rendering is not good.  It certainly *looks* like a bug.  Perhaps would be better to have some sort of icon next to each reserved unit?

**Comment by jeremystretch** [comment]
This is working as intended and was included in the v3.3 beta for review and testing. Please see #4350.
